src/applications/personalization/preferences/components/PreferenceItem.jsx

```diff
@@ -4,21 +4,50 @@ import { Link } from 'react-router';
 import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
 import AlertBox from '@department-of-veterans-affairs/formation/AlertBox';
 
+const CallToAction = ({ cta }) => {
+  const { description, link, text } = cta;
+  return (
+    <div>
+      {description && description}
+      {link &&
+        text && (
+          <Link className="usa-button" to={link}>
+            {text}
+          </Link>
+        )}
+    </div>
+  );
+};
+
+const FAQItem = ({ faq }) => {
+  const { title, component: FAQComponent } = faq;
+  return (
+    <AdditionalInfo
+      tagName={'h5'}
+      additionalClass="benefit-faq"
+      triggerText={title}
+    >
+      <FAQComponent />
+    </AdditionalInfo>
+  );
+};
+
+const FAQList = ({ faqs }) => (
+  <div>
+    {faqs.map((faq, idx) => (
+      <FAQItem faq={faq} key={idx} />
+    ))}
+  </div>
+);
+
 export default function PreferenceItem({
   handleViewToggle,
   handleRemove,
   isRemoving,
   benefit,
 }) {
-  const {
-    title,
-    introduction,
-    slug,
-    ctaLink,
-    ctaText,
-    faqTitle,
-    faqComponent: FAQComponent,
-  } = benefit;
+  const { title, introduction, slug, cta, faqs } = benefit;
+  const hasCTA = cta.description || (cta.link && cta.text);
```

Comment:
Would it make sense for the `hasCTA` condition to be moved into the `CallToAction`, since it looks like that condition also drives that component's rendering?

Reply:
Good idea, updated.